examples/cloudtrail/README.md

```diff
@@ -1,57 +1,40 @@
-# cloudtrail example
-
-Setup the application which retrieves `CloudTrail` logs from `S3` bucket and sends them to your *Coralogix* account.
-
-# Usage
-
-To run this example you need to execute:
-
+# cloudtrail
+
+Manage the application which retrieves `CloudTrail` logs from `S3` bucket and sends them to your *Coralogix* account.
+
+## Usage
+
+To run this example you need to save this code in Terraform file, and change the values according to our settings.
+
+```hcl
+terraform {
```

Comment:
This needs to be just:

provider "aws" {
}

Not the whole terraform {...} section.